hawkular-javaagent-wildfly-feature-pack/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml

```diff
@@ -33,9 +33,9 @@ storage-adapter:
   type:      HAWKULAR
   tenant-id: "${hawkular.rest.tenantId:hawkular}"
   feed-id:   "${hawkular.rest.feedId:autogenerate}"
-  url:       "${hawkular.rest.host:http://hawkular-server:8080}"
-  username:  "${hawkular.rest.user}"
-  password:  "${hawkular.rest.password}"
+  url:       "${hawkular.rest.url:http://hawkular-server:8080}"
```

Comment:
there are other places where this hawkualr.rest.host is defined that also needs to change. Since this is independent of what this PR is really for, just revert this line back to using hawkualr.rest.host and we'll fix that later.